float64 stiffness_value
---
bool result